$NetBSD: patch-ab,v 1.1.1.1 2007/04/24 16:34:02 dmcmahill Exp $

Avoid conflicting with the wcwidth which exists in libc but
with a different prototype.

--- utf8.c.orig	2007-03-05 02:15:33.000000000 +0000
+++ utf8.c
@@ -62,7 +62,7 @@ static int bisearch(ucs_char_t ucs, cons
  * in ISO 10646.
  */
 
-static int wcwidth(ucs_char_t ch)
+static int git_wcwidth(ucs_char_t ch)
 {
 	/*
 	 * Sorted list of non-overlapping intervals of non-spacing characters,
@@ -207,7 +207,7 @@ invalid:
 		return 0;
 	}
 
-	return wcwidth(ch);
+	return git_wcwidth(ch);
 }
 
 int is_utf8(const char *text)
